mysql-test/suite/galera/t/galera_insert_bulk.test

```diff
@@ -77,12 +100,23 @@ INSERT INTO t1 VALUES (1001);
 --connection node_1
 COMMIT;
 
-# Expect two keys to be added for bulk insert: DB-level shared key and table-level exclusive key.
+# Expect three keys to be added for bulk insert: "zero-level" key, DB-level shared key and table-level exclusive key.
 --let $bulk_keys_count = `SELECT VARIABLE_VALUE - $before_bulk_keys FROM INFORMATION_SCHEMA.GLOBAL_STATUS WHERE VARIABLE_NAME = 'wsrep_repl_keys'`
 --echo $bulk_keys_count
 
 --connection node_2
+
+# We need to trigger Galera-level certification conflict. For this:
+# - start applying bulk insert from the other node before local insert certifies
+# - certifying local insert will lead to the conflict
+# - keep applying bulk insert
+SET DEBUG_SYNC = 'wsrep_before_certification WAIT_FOR sync.wsrep_apply_cb_reached';
+SET DEBUG_SYNC = 'wsrep_after_certification SIGNAL signal.wsrep_apply_cb';
+
 --error ER_LOCK_DEADLOCK
```

Comment:
ok, it is clear now that conflict happens now in node_2 and apply_cb sync point in node_2 is valid. However, it is not obvious that CREATE TABLE (from node_1) avoids the sync point and can apply in node_2.
I suggest to create the table before setting applier sync point, just for clarity

Reply:
There is a separate `apply_toi()` method that has its own `sync.wsrep_apply_toi` sync point. But yes, I agree it's less confusing to pause applying after `CREATE TABLE`.